src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -149,60 +151,33 @@ void resolveKeyToPropertyWithAPropertyWithoutAnObject() {
         verify(dummy).hello();
     }
 
-    @Test
-    void resolveKeyToPropertyWithAPropertyWithAnObject() {
-        // given
-        final Superhero person = mock(Superhero.class);
-        final DummyService dummy = mock(DummyService.class);
-        doReturn(person).when(mockedFaker).superhero();
-        doReturn("Luke Cage").when(person).name();
-
-        // when
-        final String actual = fakeValuesService.resolve("property.advancedResolution", dummy, mockedFaker, mockedFaker.getContext());
-
-        // then
-        assertThat(actual).isEqualTo("Luke Cage");
-        verify(mockedFaker).superhero();
-        verify(person).name();
-    }
-
     @Test
     void resolveKeyToPropertyWithAList() {
         // property.resolutionWithList -> #{hello}
         // #{hello} -> DummyService.hello
-
```

Comment:
mocked tests is painful to maintain
this one is rewritten in a way without mocks